test/js/bun/resolve/import-meta.test.js

```diff
@@ -162,3 +162,7 @@ it('require("bun") works', () => {
 it('import("bun") works', async () => {
   expect(await import("bun")).toBe(Bun);
 });
+
+it("require.resolve with empty options object", () => {
+  expect(require.resolve("bun", {})).toBe("bun");
```

Comment:
require.resolve may happen at build-time 

instead, can you make this something which is definitely not statically analyzable?

```suggestion
  expect(require.resolve(import.meta.path, {})).toBe(import.meta.path);
```